### PR TITLE
Fix guillotine buckling

### LINKED ISSUE #53869: Guillotines do not allow you to buckle people to them
## Testmerges:
#53329: 'testing smooth movement fixes' by actioninja at commit e52a1bd8dd
#53596: 'Removes Admin Stickyban Exemption Workaround' by bobbahbrown at commit adbee092e4

## Reproduction:

Spawn a guillotine and 2 humans, occupy one of the humans. It is impossible to buckle the other human to a guillotine unless they are on the same tile. Guillotines do not allow mobs to share the same tile, so it is impossible to buckle anyone to a guillotine

### PR DESCRIPTION
**Pull request by Yenwodyah**
## About The Pull Request

Buckling code changed and made you unable to stick people in guillotines. this fixes it.
fixes #53869

## Changelog
:cl:
fix: You can buckle people to guillotines again
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
Can you explain how? And if so, put it in a comment? This just looks like redundant code at first glance.

**Comment by Timberpoes** [comment]
> 
> 
> Can you explain how? And if so, put it in a comment? This just looks like redundant code at first glance.

Looks like he's setting a default arg of check_loc = FALSE. The parent calls all have a default value of check_loc = TRUE. I'm gonna assume you can't be buckled to the gullotine because you don't share a loc with it? I haven't seen one in-game for months.

**Comment by Jared-Fogle** [comment]
If that's the case, the code should read `..(check_loc = FALSE)` to be obvious.

**Comment by Yenwodyah** [comment]
Simplified things a little bit. Now it just needs to override one proc. Also fixed a grammar error.

File: code/game/objects/structures/guillotine.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 		to_chat(usr, "<span class='warning'>You need to raise the blade before buckling someone in!</span>")
 		return FALSE
 
-	return ..(M, force, FALSE)
+	return ..(M, force, FALSE) //check_loc = FALSE to allow moving people in from adjacent turfs
```
Comment (Jared-Fogle) [inline code/game/objects/structures/guillotine.dm:205]:
```suggestion
	return ..(M, force, check_loc = FALSE) //check_loc = FALSE to allow moving people in from adjacent turfs
```

**Pull request merged**